src/Sfx/App/Scripts/Models/DataModels/Collections.ts

```diff
@@ -270,6 +270,15 @@ module Sfx {
             return [allNodes, seedNodes].concat(Object.keys(counts).map(key => counts[key]));
         }
 
+        public setAdvancedMode(state: boolean): void {
+            this.collection.forEach( node => {

```

Comment:
nit: I would consider moving this loop to its own function, so that it becomes clearer that advanced mode is really a collection different things which can be set to advanced, rather than only the nodes. That can be done later too